### ISSUE TITLE
Money

### ISSUE THREAD

**Issue by btwentyone** [TanguyOrtegat/esx_jb_advanced_truck_inventory#12]
Be able to deposit money in the trunk of the car, will not hurt for the roleplay

**Comment by TanguyOrtegat** [comment]
you can alreay put it in bank...

**Comment by btwentyone** [comment]
Yes of course but for more roleplay it will be good to be able to deposit in the trunk of the vehicle.
No way to steer, etc.
Especially for Brink's
